### DISCUSSION TITLE
Hidiffusion - how to use ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#3446] [Q&A] [answered]
How do one use Hidiffusion in Text2Image tab?

Do I just enabled it and enter a high value in width and height?

I tried that but got out an  of vram error.

**Comment by vladmandic** [comment]
hidiffusion does not reduce memory consuption, it allows models trained on lower resolutions to provide somewhat sane results on higher resolutions where otherwise they would be a nightmare.

to reduce memory usage, look at offloading options.

**Reply by cgidesign-de** [reply]
Ok.
But how about img2img?

Does HiDiffusion need Resize?
![HiDiffusion](https://github.com/user-attachments/assets/4b9dd9fb-9bc4-4d15-9923-5b30eb18d43b)

**Reply by vladmandic** [reply, marked as answer]
hidiffusion works regardless if its first or second pass and works on whatever is the generate size. 
which can be either original size (in first pass) or resized size (in second pass). if you don't have resize enabled, then resize size is irrelevant.

**Reply by cgidesign-de** [reply]
Thanks. So I did it right - just need to experiment more :-)
